iml-manager-cli/src/main.rs

```diff
@@ -148,6 +151,22 @@ struct CmdWrapper {
     command: Command,
 }
 
+#[derive(serde::Serialize, serde::Deserialize, Debug, Clone)]
+#[serde(rename_all = "snake_case")]
+enum RunStratagemCommandResult {
```

Comment:
Why does this not use the Error trait?

Reply:
I'll add

Reply:
This should be nested inside `ImlManagerClientError` as well.